spec/mailers/received_kudos_mailer_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe ReceivedKudosMailer, :type => :mailer do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
imo this file should either (a) have specs, (b) reference a ticket to add specs in the future, or (c) be deleted.

Testing emails might be a chunk of work, so it's okay with me if that's outside the scope of this PR (and I'd prefer option b)